intelmq/bots/outputs/smtp_batch/output.py

```diff
@@ -253,8 +252,11 @@ def prepare_mails(self):
             fieldnames = set()
             rows_output = []
             for row in lines:
-                if threshold and row["time.observation"][:19] < threshold.isoformat()[:19]:
-                    continue
+                try:
+                    if threshold and row["time.observation"][:19] < threshold.isoformat()[:19]:
+                        continue
+                except KeyError:
+                    self.logger.warning(f"Warning: %s row skipped due to time.observation error", mail_record)
```

Comment:
I suggest to mention what the error is (missing field). And format it with repr
```suggestion
                    self.logger.warning(f"Warning: row skipped because it is missing the field 'time.observation': %r", mail_record)
```

Reply:
Right, I've unified it for every logger command